include/aikido/planner/vectorfield/VectorFieldPlanner.hpp

```diff
@@ -0,0 +1,87 @@
+#ifndef AIKIDO_PLANNER_VECTOR_FIELD_PLANNER_HPP_
+#define AIKIDO_PLANNER_VECTOR_FIELD_PLANNER_HPP_
+
+#include <boost/function.hpp>
+#include <aikido/constraint/Testable.hpp>
+#include <aikido/statespace/dart/MetaSkeletonStateSpace.hpp>
+#include <aikido/trajectory/Spline.hpp>
+
+namespace aikido {
+namespace planner {
+namespace vectorfield {
+
+struct VectorFieldPlannerStatus
+{
+  enum Enum
+  {
+    TERMINATE,
+    CACHE_AND_TERMINATE,
+    CACHE_AND_CONTINUE,
+    CONTINUE
+  };
+};
```

Comment:
If you put this enum in a struct is to avoid scope confliction, you could consider using [`enum class`](http://en.cppreference.com/w/cpp/language/enum), which is introduced in C++11.